guides/release/testing/testing-components.md

```diff
@@ -218,21 +218,21 @@ passing along the form's data:
 > component comment-form`.
 
 ```javascript {data-filename="app/components/comment-form.js"}
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  comment: '',
+export default class CommentFormComponent extends Component {
+  comment = '';
 
-  actions: {
-    submitComment() {
-      this.submitComment({ comment: this.comment });;
-    }
-  }
+  @action
+  submitComment() {
+    this.submitComment({ comment: this.comment });;
```

Comment:
On glimmer components, the passed in arguments are not on `this`, they are in `this.args`.

```suggestion
    this.args.submitComment({ comment: this.comment });;
```